src/Sidebar.js

```diff
@@ -11,7 +11,7 @@ import AppsIcon from "@material-ui/icons/Apps";
 import ExpandLessIcon from "@material-ui/icons/ExpandLess";
 import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import AddIcon from "@material-ui/icons/Add"
-import {collection , getDocs , doc} from "firebase/firestore/lite"
+import { collection, getDocs } from "firebase/firestore"
```

Comment:
This was the fix of bug 1. Always read the documentation to understand how a lib is used.

Reply:
I tried my best it's just I got confused with reading a lot of documentation (StackOverflow) so I end up with "lite" in the path. I looked at Firebase source documentation you are right it was my bad. Still learning to look into getting and following the right documentation. Hopefully getting better :-)